src/xrc/xmlres.cpp

```diff
@@ -1852,6 +1852,52 @@ bool GetStockArtAttrs(const wxXmlNode *paramNode,
     return false;
 }
 
+// Load a bitmap from a file.
+wxBitmap LoadBitmapFromFile(wxXmlResourceHandlerImpl* impl,
```

Comment:
This is not necessarily a file, is it? It's `wxFileSystem` element, so maybe rename it like this
```suggestion
wxBitmap LoadBitmapFromFS(wxXmlResourceHandlerImpl* impl,
```
?